src/settings/LegalScreen.js

```diff
@@ -18,19 +18,35 @@ type Props = $ReadOnly<{|
 /** (NB this is a per-account screen: it leads to this realm's policies.) */
 export default function LegalScreen(props: Props): Node {
   const realm = useSelector(getRealmUrl);
+  const realmName = useSelector(getRealmName);
 
-  const openTermsOfService = useCallback(() => {
-    openLinkEmbedded(new URL('/terms/?nav=no', realm).toString());
-  }, [realm]);
+  const openZulipPolicies = useCallback(() => {
+    openLinkEmbedded('https://zulip.com/policies/?nav=no');
+  }, []);
 
-  const openPrivacyPolicy = useCallback(() => {
-    openLinkEmbedded(new URL('/privacy/?nav=no', realm).toString());
+  const openRealmPolicies = useCallback(() => {
+    openLinkEmbedded(new URL('/policies/?nav=no', realm).toString());
   }, [realm]);
 
   return (
     <Screen title="Legal">
-      <NestedNavRow label="Terms of service" onPress={openTermsOfService} />
-      <NestedNavRow label="Privacy policy" onPress={openPrivacyPolicy} />
+      <NestedNavRow label="Zulip terms" onPress={openZulipPolicies} />
+      <NestedNavRow
+        // These are really terms set by the server admin responsible for
+        // hosting the org, and that server admin may or may not represent
+        // the org itself (e.g., orgs on Zulip Cloud), as this text might be
+        // read to imply. But:
+        // - We don't want to complicate the wording. Not everyone knows
+        //   what a server is.
+        // - These terms will often differ from Zulip's own terms (the ones
+        //   at the other link), and we have to include them.
+        // - These terms will apply to all users in the org, in all cases.
+        label={{
+          text: 'Terms for {realmName}',
+          values: { realmName: <RawLabel style={{ fontWeight: 'bold' }} text={realmName} /> },
```

Comment:
Neat!

From that error message, it looks like the relevant bit of the libdef is here:
```js
  declare export interface IntlFormatters<T = React$Node, R = T> {
// …
    formatMessage(
      descriptor: MessageDescriptor,
      // The `+` was added to make the properties covariant rather
      // than invariant, something TypeScript can't do.
      values?: {| +[key: string]: PrimitiveType | FormatXMLElementFn<string, string> |},
    ): string;
    formatMessage(
      descriptor: MessageDescriptor,
      // The `+` was added to make the properties covariant rather
      // than invariant, something TypeScript can't do.
      values?: {| +[key: string]: PrimitiveType | React$Node | FormatXMLElementFn<T, R> |},
    ): React$Node;
```

So there's an overload -- two signatures for the same method `formatMessage`. Up to now we've used the first one, and now we want to use the second one.

I suspect a key piece of why there's an error here is that, via our `GetText` type, we're saying that our `formatMessage` call (in `TranslationProvider.js`) should return string. From examining those overloaded definitions, one can see that that's not what `formatMessage` returns anymore if you pass it a value that's a `React$Node` -- in that case it returns another `React$Node`.

I'll fiddle with it a bit and see what I find.